ibt/up_command.py

```diff
@@ -64,8 +64,8 @@ def run(self, ctx, project, args):
                 uid, group_name, gid, user_name = ctx.user_info()
                 with open(os.path.join(dir, "Dockerfile"), "wt") as f:
                     f.write("FROM {}\n".format(docker_image))
-                    f.write("RUN groupadd -g {} {}\n".format(gid, group_name))
-                    f.write("RUN useradd -l -u {} -g {} {}\n".format(uid, gid, user_name))
+                    f.write("RUN groupadd -g {} \"{}\"\n".format(gid, group_name))
+                    f.write("RUN useradd -l -u {} -g {} \"{}\"\n".format(uid, gid, user_name))
```

Comment:
If it works, it would probably be better to use `pipes.quote`, instead of wrapping with double quotes. e.g.:

```
import pipes
```
...
```
f.write("RUN groupadd -g {} {}\n".format(pipes.quote(gid), pipes.quote(group_name)))
```

What do you think?